holoviews/tests/plotting/matplotlib/testutils.py

```diff
@@ -30,7 +30,7 @@ def test_polygon_to_path_patches(self):
             (1.5, 2), (2, 3), (1.6, 1.6), (1.5, 2),
             (2.1, 4.5), (2.5, 5), (2.3, 3.5), (2.1, 4.5)]))
         self.assertEqual(paths[0][0].get_path().codes, np.array([1, 2, 2, 79, 1, 2, 2, 79, 1, 2, 2, 79], dtype='uint8'))
-        self.assertEqual(paths[0][1].get_path().vertices, np.array([(6, 7), (7, 5), (3, 2), (6, 7)]))
+        self.assertEqual(paths[0][1].get_path().vertices, np.array([(3, 2), (7, 5), (6, 7),  (3, 2),]))
```

Comment:
I assume this existing test was updated due to some bug that was fixed? i.e was the original test incorrect?

Reply:
Spatialpandas will reorient vertices to be in the expected order, so to ensure that tests will work across all interfaces I need to make sure that the test data is oriented correctly. Otherwise spatialpandas will have a different orientation to other interfaces on comparison.

Reply:
Ok, that makes sense. Thanks for the explanation.